src/templates/issues.handlebars

```diff
@@ -48,13 +36,11 @@
                          opened {{timeago created_at false}} ago
 	                 </span>
 	              </div>
-                  <div class="t-xxxs r-iblock">
-                     {{#each labels}}
-                         {{#eq (slug name) 'priorityhigh'}}
-                             <div class="r-iblock btn" style="border-color:#{{color}}; color:#{{color}};">{{name}}</div>
-                         {{/eq}}
-                     {{/each}}
-                  </div>
+                  {{#if high_priority}}
```

Comment:
you have full control over the data, so there has to be a better way than asking a question in the template